geocoding/geocode.py

```diff
@@ -112,14 +102,14 @@ def geocode(data_file, api_key, use_column="Address"):
         api_key (str): API key for Google Map Geocoding API
         use_column (str, optional): The column that is to be used as input to API while geocoding. Defaults to "Address".
     """
-    make_dir()
+    (pathlib.Path.cwd() / "Sheets").mkdir(parents=True, exist_ok=True)
     gmaps = googlemaps.Client(key=api_key)
 
     if data_file.endswith(".json"):
         geocode_json(data_file, gmaps, use_column)
     else:
```

Comment:
```suggestion
    elif data_file.endswith(".xlsx") or data_file.endswith(".xls"):
```
Explicit is better than implicit.

Reply:
Added the change.